lib/containers/runtime.pm

```diff
@@ -62,24 +110,40 @@ when it exits or when the daemon exits
 sub up {
     my ($self, $image_name, %args) = @_;
     die 'image name or id is required' unless $image_name;
-    my $mode           = $args{daemon}         ? '-d'    : '-it';
-    my $remote         = $args{cmd}            ? 'sh -c' : '';
-    my $keep_container = $args{keep_container} ? ''      : '--rm';
-    my $ret            = $self->_rt_script_run(sprintf qq(run %s %s %s %s '%s'), $keep_container, $mode, $image_name, $remote, $args{cmd});
-    record_info "Remote run on $image_name", "options $keep_container $mode $image_name $remote $args{cmd}";
+    my $mode           = $args{daemon}         ? '-d'                    : '-i';
+    my $remote         = $args{cmd}            ? "-- sh -c '$args{cmd}'" : '';
+    my $name           = $args{name}           ? "--name $args{name}"    : '';
+    my $keep_container = $args{keep_container} ? ''                      : '--rm';
+    my $params         = sprintf qq(%s %s %s), $keep_container, $mode, $name;
+    my $ret            = $self->_rt_script_run(sprintf qq(run %s %s %s), $params, $image_name, $remote, timeout => $args{timeout});
+    record_info "Remote run on $image_name", "options $keep_container $mode $name $image_name $remote";
```

Comment:
```suggestion
    record_info "INFO", "Remote run on $image_name:\noptions $keep_container $mode $name $image_name $remote";
```

Reply:
i know and i tried to keep it short. I will try something like `cmd_info`

Reply:
"Remote run on" is short, but $image_name can make the string very long...  `cmd_info` or similar sounds good.